src/cc65/codeent.h

```diff
@@ -151,6 +151,15 @@ INLINE CodeLabel* CE_GetLabel (CodeEntry* E, unsigned Index)
 #  define CE_GetLabel(E, Index) CollAt (&(E)->Labels, (Index))
 #endif
 
+#if defined(HAVE_INLINE)
+INLINE void CE_ReplaceLabel (CodeEntry* E, CodeLabel* L, unsigned Index)
```

Comment:
All other CE_ "entries" hav a brief comment. I'd appreciate sticking to that here too.

Reply:
Done.